ansible_base/feature_flags/views.py

```diff
@@ -7,6 +9,7 @@
 from .utils import get_django_flags
 
 
+@extend_schema(request=None, responses=OpenApiTypes.OBJECT, examples=[OpenApiExample(name="featureflags", value={"FLAG1": True, "FLAG2": False})])
```

Comment:
Can we be sure that drf_spectacular is installed at all times?

Reply:
ie: we may want to have our own decorator util in DAB to check `INSTALLED_APPS` for `ansible_base.api_documentation` before importing drf_spectacular and trying to extend the schema.

Reply:
Yeah, that's a good idea.  It's meant to be optional to use api_documentation so anything that would be loaded if that app isn't loaded should probably have a check.  We have a couple of other places like that too, but I'll see if I can find a nice way to handle that.

Reply:
https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/api_documentation/customizations.py has a different approach, where I'm guessing we can extend the view schemas as well. 

I'm guessing we'd want to keep the schema extensions where the view is defined though.

Reply:
Those are normally intended to add documentation to things defined outside the code (libraries, dependencies etc.) but seems reasonable to use for the purpose of loading only if docs are enabled.  It doesn't solve the import issue, but I switched to that format to match what was done in https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/jwt_consumer/eda/auth.py and so that the view is only modified when docs are enabled.